libs/game/multiplayer.ts

```diff
@@ -0,0 +1,12 @@
+namespace multiplayer {
+    // registration code should probably go in here, but it's giving an error so moving to startup.ts for now
```

Comment:
Ah, this is where I had the code that is over in pxt-arcade originally but it was the issue I mentioned with dependencies -- there's a decently good chance that you can remove all the changes from pxt-arcade & uncomment here and it should work just fine now that it is in game. (though we should either register this in an init function or have the sim side send an event that enables this when in multiplayer mode)